packages/design-system/ui/src/components/code-block/code-block.tsx

```diff
@@ -141,10 +141,39 @@ const Body = ({
               backgroundColor: "#111827",
             },
             styles: [
+              ...themes.palenight.styles,
               {
                 types: ["keyword"],
                 style: {
-                  color: "var(--fg-on-color)",
+                  fontStyle: "normal",
```

Comment:
very-nit and non-blocing: Noticed we're mixing rgb and hex. Do we want to be consistent?